beacon-chain/db/filesystem/blob.go

```diff
@@ -241,22 +241,26 @@ func (bs *BlobStorage) Prune(pruneBefore primitives.Slot) error {
 
 	log.Debug("Pruning old blobs")
 
-	folders, err := afero.ReadDir(bs.fs, ".")
-	if err != nil {
-		return err
-	}
 	var totalPruned int
-	for _, folder := range folders {
-		if folder.IsDir() {
-			num, err := bs.processFolder(folder, pruneBefore)
+	err := afero.Walk(bs.fs, ".", func(path string, info os.FileInfo, err error) error {
```

Comment:
Could we add errors.Wrap to add more context about where specifically this failed?